lib/utils.pm

```diff
@@ -765,6 +766,24 @@ sub assert_screen_with_soft_timeout {
     return assert_screen $mustmatch, $args{timeout} - $args{soft_timeout};
 }
 
+=head2 add_serial_console
+
+    add_serial_console($console);
+
+Adds $console to /etc/securetty (unless already in file), enables systemd
+service and start it.
+
+=cut
+sub add_serial_console {
+    my ($console) = @_;
+    my $service   = 'serial-getty@$console';
+    my $config    = '/etc/securetty';
+    my $name      = "/dev/$console";
+    type_string("grep -q '^$console\$' $config || echo '$console' >> $config\n");
```

Comment:
This is risky if we are already on a fast console (i.e. the virtio serial console) because we may start sending 'systemctl enable...' before the shell takes back control of stdio from echo. Although common case will be on VNC where I don't think we have that problem.